venter/admin.py

```diff
@@ -115,9 +115,21 @@ def send_emails(modeladmin, request, queryset):
             # Retrieves a list of email ids for the selected authorities. Excludes the authority bodies with no email id
             recipient_list = queryset.values_list('authorities__email', flat=True).exclude(authorities__email=None)
 
+            authority = item.authorities.values_list('name', flat=True)
+            auth = ', '
+            auth = auth.join(authority)
+
             # Composes the email to be sent to the authorities and sends it to the recipients
             send_mail(subject, message, sender_id, recipient_list)
 
+            # check if email_sent_to list is empty or not, if empty assign auth value else append auth value
+            if item.email_sent_to:
+                item.email_sent_to = item.email_sent_to + ', ' + auth
+            else:
+                item.email_sent_to = auth
+
+            item.authorities.clear()
```

Comment:
Why clear authorities?

Reply:
So that it doesn't show the names authorities to whom the mail has been sent to in the pending list

Reply:
Ah, right this is a list of just _pending_ authorities now.